monitoring/monitoring.go

```diff
@@ -0,0 +1,61 @@
+package monitoring
+import (
+	"google.golang.org/grpc/codes"
+)
+
+
+type RpcType string
```

Comment:
I realize that you are creating this as a rough cut for a proposal, but perhaps give `golint` a quick run to bring everything to an equal level of polish.